applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py

```diff
@@ -0,0 +1,96 @@
+        self.eigenvaluevariable = getattr(StructuralMechanicsApplication,  "EIGENVALUE_VECTOR")
+        self.eigenvectorvariable = getattr(StructuralMechanicsApplication, "EIGENVECTOR_MATRIX")
+        self.dof_variable_name = KratosMultiphysics.KratosGlobals.GetVariable( settings["dof_variable_name"].GetString()) 
+        self.animation_steps =  settings["animation_steps"].GetInt()
+        
+        #self.dim = self.model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE]
+                                                                              
+    def ExecuteInitialize(self):
+        pass
+    
+    def ExecuteBeforeSolutionLoop(self):
+        pass
+    
+    def ExecuteInitializeSolutionStep(self):
+        pass
+
+    def ExecuteFinalizeSolutionStep(self):
+        pass
+              
+    def ExecuteBeforeOutputStep(self):
+        pass
+
+    def ExecuteAfterOutputStep(self):
+        pass
+
+    def ExecuteFinalize(self):
+        gid_mode = KratosMultiphysics.GiDPostMode.GiD_PostBinary
+        singlefile = KratosMultiphysics.MultiFileFlag.SingleFile
+        deformed_mesh_flag = KratosMultiphysics.WriteDeformedMeshFlag.WriteUndeformed
+        write_conditions = KratosMultiphysics.WriteConditionsFlag.WriteConditions
+        
+        ## Debug
+        #for node in self.model_part.Nodes:
+            #EigenMatrix = node.GetValue(self.eigenvectorvariable)
+            #print(EigenMatrix)
+        
+        eigen_values = [ev for ev in self.model_part.ProcessInfo[self.eigenvaluevariable]]
+        for evs, count in zip(eigen_values, range(len(eigen_values))):
+            # We create a different ouput for each eigenvalue
+            output_file = "EigenValue_w="+str(round(evs, 3))
+            
+            gid_io = KratosMultiphysics.GidIO(output_file, gid_mode, singlefile, deformed_mesh_flag, write_conditions)
+            gid_io.InitializeMesh(0.0) 
+            gid_io.WriteMesh(self.model_part.GetMesh())
+            gid_io.WriteNodeMesh(self.model_part.GetMesh())
+            gid_io.FinalizeMesh() 
+            
+            gid_io.InitializeResults(0.0, self.model_part.GetMesh())
+            
+            for label in range(self.animation_steps):
+                angle = 2.0 * math.pi * label/self.animation_steps
+                # NOTE: This is slow, because I am accessing to the same information several times
+                for node in self.model_part.Nodes:
+                    EigenMatrix = node.GetValue(self.eigenvectorvariable)
+                    # NOTE: The DoF stored include the velocity and acceleration, solve this in the future
+                    dof_values = KratosMultiphysics.Vector(3)
+                    dof_values[0] = math.cos(angle) * EigenMatrix[count, 3]
+                    dof_values[1] = math.cos(angle) * EigenMatrix[count, 4]
+                    dof_values[2] = math.cos(angle) * EigenMatrix[count, 5]
+                    node.SetSolutionStepValue(self.dof_variable_name, 0, dof_values) 
```

Comment:
Hi @loumalouomega  and @msandre , I corrected in the postprocess file the access to the matrix since now there are only displacementes as dof's. 

I also have created a new solver for my application due to I'm not working with materials.py as in the structural.

Finally, for printing the modes I have used the utility commented by @msandre  adding the following lines in my main script;

eigen_values = [ev for ev in main_model_part.ProcessInfo[KratosStructural.EIGENVALUE_VECTOR]] 
print ("The Eigen Values are:") 
print (eigen_values) 
eigen_utility = KratosStructural.EigenvectorToSolutionStepVariableTransferUtility() 
for step in range(len(eigen_values)): 
    main_model_part.ProcessInfo[KratosMultiphysics.TIME] = float(step+1) 
    eigen_utility.Transfer(main_model_part,step,0) 
    gid_output.PrintOutput() 

In case of animation are required it is possible to use the postprocess file of @loumalouomega , so I think it is a good idea to keep it.

@loumalouomega , I think it would be useful document the way of compilation and how it is possible to use FEAST library. If you want I can prepare it.  What do you think??

Thank you for your help @msandre @loumalouomega